.azure/pipelines/ci.yml

```diff
@@ -51,7 +45,8 @@ stages:
 
   - stage: backend_build
```

Comment:
Check if a failure in _frontend_e2e_ quits the continuation of the steps.

Reply:
Naaaa, it continues:
![grafik](https://user-images.githubusercontent.com/5755834/165372482-375db942-f981-492d-af6a-79463d62ea60.png)

There's no dependency so why should it now anyway. Need to configure or move the dependencies around. I'll do the latter as outlined in https://github.com/gfoidl-Tests/Vue_Server_Test/pull/9